lib/tasks/anonymize.rake

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+namespace :anonymize do
```

Comment:
Namespace must be consistent with the rest of the tasks in this gem: `cdtb:anonimize:...`